recipes/git-dwim

```diff
@@ -1 +1 @@
-(git-dwim :fetcher wiki)
+(git-dwim :repo "rubikitch/emacs-git-dwim" :fetcher github)
```

Comment:
Yes, that should be removed. It's personal configuration which shouldn't be distributed.